source/class/qx/tool/cli/commands/Serve.js

```diff
@@ -112,7 +112,7 @@ qx.Class.define("qx.tool.cli.commands.Serve", {
 
       const app = express();
       const website = new qx.tool.utils.Website();
-      if (apps.length === 1 && apps[0].getWriteIndexHtmlToRoot() && this.argv.showStartpage === false) {
+      if (((apps.length === 1) || apps[0].getWriteIndexHtmlToRoot()) && (this.argv.showStartpage === false)) {
```

Comment:
this isn't quite right - it may be `apps[1]` that has `.writeIndexHtmlToRoot` set